src/components/Dropdown/Dropdown.tsx

```diff
@@ -121,7 +121,7 @@ export function Dropdown<V>({
 
     const menuProps = getMenuProps({
         onBlur: onMenuBlur,
-        onFocus: onMenuFocus,
+        onClick: onMenuFocus,
```

Comment:
this looks a bit confusing, is it intentional?

Reply:
It was not final result i just check test cases